.pre-commit-config.yaml

```diff
@@ -9,6 +9,7 @@ repos:
       - id: check-toml
       - id: end-of-file-fixer
       - id: trailing-whitespace
+        exclude-types: ["txt","tsv"]
```

Comment:
I had to add this line to prevent the pre-commit hook to trim whitespaces at the end of lines on this expected result file.
If I the end-of-line whitespaces are removed, the test fails